src/components/v5/shared/Navigation/ColonySwitcher/partials/TitleSections/NoColoniesJoinedSection.tsx

```diff
@@ -0,0 +1,31 @@
+import { SmileyMelting } from '@phosphor-icons/react';
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { formatText } from '~utils/intl.ts';
+
+import { BaseTitleSection } from './BaseTitleSection.tsx';
+
+const displayName =
+  'v5.common.Navigation.ColonySwitcher.partials.NoColoniesJoinedSection';
+
+const MSG = defineMessages({
+  noColoniesJoinedSectionTitle: {
+    id: `${displayName}.noColoniesJoinedSectionTitle`,
+    defaultMessage: 'No colonies joined',
+  },
+  noColoniesJoinedSectionDescription: {
+    id: `${displayName}.noColoniesJoinedSectionDescription`,
+    defaultMessage: 'Once you join or create a colony, they will appear here.',
+  },
+});
+
+export const NoColoniesJoinedSection = () => {
```

Comment:
```suggestion
export const EmptyJoinedColoniesSection = () => {
```
It’s just a suggestion - feel free to choose a name that works best for you. From my perspective, this could be more understandable 🤗

Reply:
Hey @Nortsova thank you for pointing this out! ❤️ 
I renamed it to `EmptyJoinedColoniesSection` as it definitely is more clear